src/pspm_cfg/pspm_cfg_glm_hp_rew.m

```diff
@@ -0,0 +1,50 @@
+function [glm_hp_rew] = pspm_cfg_glm_hp_rew
+% GLM HP FC
+
+% Initialise
+global settings
+if isempty(settings), pspm_init; end;
+
+% set variables
+
+vars = struct();
+vars.modality = 'HPR';
+vars.modspec = 'hp_fc';
+vars.glmref = { ...
+        ['Xia Y, Liu H, Kälin OK, Gerster S, Bach DR (under review). Measuring', ...
+         'human Pavlovian appetitive conditioning and memory retention.']};
+vars.glmhelp = '';
+
+% load default settings
+glm_hp_rew = pspm_cfg_glm(vars);
+
+% set correct name
+glm_hp_rew.name = 'GLM for HP (reward-conditioning)';
+glm_hp_rew.tag = 'glm_hp_rew';
+
+% set callback function
+glm_hp_rew.prog = @pspm_cfg_run_glm_hp_rew;
+
+%% Basis function
+% HPRF
+hprf_rew        = cfg_const;
+hprf_rew.name   = 'HPRF_REW';
+hprf_rew.tag    = 'hprf_rew';
+hprf_rew.val    = {1};
+hprf_rew.help   = {'HPRF_REW without derivatives.'};
```

Comment:
I suggest to include the full name of "HPRF_REW" in the help text for people like me who do not have knowledge of the field but may have the intention to use it as a black box for own purposes.